reconcile/test/test_terraform_cloudflare_resources.py

```diff
@@ -104,3 +114,165 @@ def external_resources(provisioner_config):
             ),
         ],
     )
+
+
+@pytest.fixture
+def mock_gql(mocker):
+    mocker.patch("reconcile.terraform_cloudflare_resources.gql", autospec=True)
+
+
+@pytest.fixture
+def mock_vault_secret(mocker):
+    mocked_vault_secret = mocker.patch(
+        "reconcile.terraform_cloudflare_resources.get_app_interface_vault_settings",
+        autospec=True,
+    )
+    mocked_vault_secret.return_value = AppInterfaceSettingsV1(vault=False)
+
+
+@pytest.fixture
+def mock_cloudflare_accounts(mocker):
+    from reconcile.gql_definitions.terraform_cloudflare_resources.terraform_cloudflare_accounts import (
+        CloudflareAccountV1 as CFAccountV1,
+    )
+    from reconcile.gql_definitions.terraform_cloudflare_resources.terraform_cloudflare_accounts import (
+        TerraformCloudflareAccountsQueryData,
+    )
```

Comment:
Any reason to reimport these?

Reply:
Yes. They have different paths. The ones on the top of the file are from `terraform_cloudflare_resources` they are slightly different.